statemachines/pre_flight_checks/test_dns

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
+DIR=`dirname $0`
+. /etc/restatemachine/shell_helpers.inc
+
+INPUT=`cat`
+Error () {
+        echo -n "$INPUT"
+        restatemachine_transition_to "test_dns"
+        restatemachine_transition_after_seconds "60"
+        restatemachine_status_message "$1"
+        exit 0
+}
+
+HOSTNAME=`echo "$INPUT" | jq -r '.hostname'`
```

Comment:
Added a small note in start, the plan is to install all dependencies including restate-machine with the gui install script. Will get to that shortly